packages/config-plugins/src/android/Manifest.ts

```diff
@@ -180,18 +180,21 @@ export function getMainActivity(androidManifest: AndroidManifest): ManifestActiv
 export function addMetaDataItemToMainApplication(
   mainApplication: ManifestApplication,
   itemName: string,
-  itemValue: string
+  itemValue: string,
+  itemType: 'resource' | 'value' = 'value'
```

Comment:
Probably want this to be more agnostic, maybe a different method could implement this, then `addMetaDataItemToMainApplication` calls into that method so the changes aren't breaking.

Reply:
since `itemType` defaults to `value` if it's not provided (this is what all the other instances of `addMetaDataItemToMainApplication` expect), this change shouldn't be breaking

you mean to make the `itemType` argument more agnostic? eg:

```suggestion
  itemType?: string
): ManifestApplication {
  itemType = itemType ?? 'value';
```